crates/forge/tests/integration/gas.rs

```diff
@@ -369,9 +369,9 @@ fn contract_poseidon_cost() {
     let result = run_test_case(&test);
 
     assert_passed!(result);
-    // 1101 = cost of deploy (see snforge_std_deploy_cost test)
+    // 1102 = cost of deploy (see snforge_std_deploy_cost test)
     // 13 = cost of 80 poseidon builtins
-    assert_gas!(result, "contract_poseidon_cost", 1101 + 13);
+    assert_gas!(result, "contract_poseidon_cost", 1102 + 13);
```

Comment:
This is strange. The cost of deploy should not change (it doesn't in the `snforge_std_deploy_test)

Reply:
Actually it changed in `snforge_std_deploy_test`. For example in `multiple_storage_writes_cost` `n_steps` goes from `477` to `562` and `range_check_builtin` from `19` to `20`. That's give us `0.525` more gas according to [this](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#calculation_of_computation_costs). Probably in other tests it's rounded as gas is `u128` and not some floating point type.

Reply:
So, if the range_check builtin number increased here too (I assume), we should increase the `13` value not the cost of deployment. `1101` stands for onchain data used by the deploy

Reply:
.After further investigation i realised that `steps` are now main factor here not `poseidon`. Increased `poseidon` number to exceed `steps` gas.